src/include/sound/routines.rb

```diff
@@ -992,7 +995,7 @@ def InstallFonts(symlink, dontask)
           res = Convert.to_map(
             SCR.Execute(
               path(".target.bash_output"),
-              Ops.add(Ops.add(Directory.ybindir, "/copyfonts "), mpoint),
+              "#{File.join(Directory.ybindir, "/copyfonts")} #{mpoint.shellescape}",
```

Comment:
NP: I'm thinking whether we should extend all `Directory` methods to accept a parameter, like `Directory.ybindir("copyfonts")` which would build the whole path. To make it more DRY...

Reply:
maybe with more friendly name like we already have for path. Also problem with that ybindir is that it does not respect Y2DIR. So I prefer stuff like find_bin("copyfonts") which will return path or raise error.